include/pokemon_storage_system.h

```diff
@@ -6,6 +6,7 @@
 #define IN_BOX_COLUMNS          6 // Number of columns, 5 Pokémon per column
 #define IN_BOX_COUNT            (IN_BOX_ROWS * IN_BOX_COLUMNS)
 #define BOX_NAME_LENGTH         8
+#define MAX_FUSION_STORAGE      5
```

Comment:
`FusionStorageIndex` doesn't return 4, so there's no point on reserving space for a 5th slot.